ruled_labels/[DRAFT]specs_monorepo.yaml

```diff
@@ -0,0 +1,160 @@
+
+name: Monorepo labels
+description: Import on 2023-05-09
+
+codes:
+  - code: A
+    name: Action 
+    description: used with GHA and trigger a certain process
+  - code: B
+    name: Breaking changes 
+    description: Upgrade dependencies, relevant for downstream Parachains/Parathreads.
+  - code: C
+    name: Complexity 
+    description: Complexity of the issues, also used for contribution guidelines.
+  - code: D
+    name: Auditing 
+    description: Required when there were changes in the paths [CI/CD INPUT NEW FILE PATHS]. Also used to indicate the need for a documentation audit of DevRel.
+  - code: I
+    name: Issues 
+    description: Categorisation of issues
+  - code: T
+    name: Topics
+    description: PR/Issue is related to this topic
```

Comment:
These are also basically release notes things.

Reply:
right now they are tighten to the release notes, but in future they won't
it's currently also used by teams to distinguish what the issue/PR is about without needing to open it (which is currently also the problem with the release notes, bc it only let's you use one topic and the teams would like to add multiple) so I think it's important to keep them

Reply:
In FRAME we use the `T1-runtime` label a lot, it is used as Joyce suggests and also by bots. We need this label or a similar one